Documentation/api-reference.md

```diff
@@ -66,11 +61,6 @@ Consent.getConsents(new AdobeCallback<Map<String, Object>>() {
 
 #### Kotlin
```

Comment:
We should group by Syntax and Example

Sytax
java

Example
java
kotlin

Reply:
but the syntax is for Java, not for Kotlin, right?

Reply:
yes

Reply:
Let follow below syntax which is the same for adobe io.
Java
Syntax
Example

Kotlin
Example